js/src/components/contentAnalysis/ReadabilityAnalysisLegacy.js

```diff
@@ -0,0 +1,58 @@
+/* global wpseoPostScraperL10n wpseoTermScraperL10n */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+
+import Results from "./Results";
+
+let localizedData = {};
+if( window.wpseoPostScraperL10n ) {
+	localizedData = wpseoPostScraperL10n;
+} else if ( window.wpseoTermScraperL10n ) {
+	localizedData = wpseoTermScraperL10n;
+}
+
+/**
+ * Redux container for the readability analysis.
+ */
+class ReadabilityAnalysis extends React.Component {
+	render() {
+		return (
+			<Results
+				canChangeLanguage={ ! ( localizedData.settings_link === "" ) }
```

Comment:
This might brake when you are not on a post or tem page, for example the author page. This is because of the key is not present in the empty object.